### PR TITLE
new package R2R-1.0.6

### PR DESCRIPTION
**Pull request by marcom**
R2R is a package for drawing RNA secondary structures.

https://sourceforge.net/projects/weinberg-r2r/
https://pubmed.ncbi.nlm.nih.gov/21205310/

### CODE REVIEW

File: R/R2R/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+for f in src/SymbolicMath.cpp \
+         src/SymbolicMath.h \
+         src/GSCConsensus.cpp \
+         src/ParseOneStockholm.cpp \
+         src/PositionBackbone_MultiStemCircularSolver.cpp \
+         src/RnaDrawer.cpp;
+do
+    cp "$f" "$f".bak
+    tr -d '\015' < "$f".bak > "$f"
+done
```
Comment (giordano) [inline R/R2R/build_tarballs.jl:29]: https://github.com/JuliaPackaging/Yggdrasil/pull/5133#issuecomment-1180886129.  I just realised you did the same for Rscape

File: R/R2R/bundled/patches/fix-format-strings.patch [hunk 2]
```diff
@@ -0,0 +1,68 @@
+diff --git a/src/ParseOneStockholm.cpp b/src/ParseOneStockholm.cpp
```
Comment (giordano) [inline R/R2R/bundled/patches/fix-format-strings.patch:1]: Any chance to upstream these patches?  As far as I can tell they can be useful to everyone else?

File: R/R2R/build_tarballs.jl (hunk 1 above)
Comment (marcom) [reply R/R2R/build_tarballs.jl:29]: Ok, will rework the build to keep the line endings in the patches by setting .gitattributes appropriately here and in Rscape

Comment (marcom) [reply R/R2R/build_tarballs.jl]: Thanks for the gitattributes tip!

File: R/R2R/bundled/patches/fix-format-strings.patch (hunk 2 above)
Comment (marcom) [reply R/R2R/bundled/patches/fix-format-strings.patch:1]: I will send the original author an email with the patches.

**Pull request merged**